mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/InMemoryMessageIdManager.java

```diff
@@ -112,8 +113,11 @@ public void setInMailboxes(MessageId messageId, List<MailboxId> mailboxIds, Mail
             for (MailboxId mailboxId: mailboxesToAdd) {
                 MessageRange messageRange = referenceMessage.getUid().toRange();
                 mailboxManager.copyMessages(messageRange, referenceMessage.getMailboxId(), mailboxId, mailboxSession);
+                Flags flags = new Flags(referenceMessage.getFlags());
+                System.out.println("###########" + flags.contains(Flag.RECENT)); //weirdly it print false here but in the in memory integration test still work
```

Comment:
So here when doing a test in memory and when doing a move there is no recent flag but the integration test pass